### PR TITLE
out_http: json_array documentation

### PR DESCRIPTION
**Pull request by cosmo0920**
Related to https://github.com/fluent/fluentd/issues/3022 and https://github.com/fluent/fluentd/pull/2973.

### CODE REVIEW

**Comment by cosmo0920** [comment]
https://github.com/fluent/fluentd-docs-gitbook/commit/b14010a9f26a92a8b626486eb0b8e4a9baf61584 should be needed but I'm not sure https://github.com/fluent/fluentd-docs-gitbook/pull/184/commits/4e6c7de7f09ba21383a6fed535874b9651acee36 is needed for users. 
(We recommend to use inter-Fluentd communicating with in/out forward.)

File: plugins/output/http.md [hunk 1]
```diff
@@ -282,6 +292,79 @@ log line4\n
...
+  <parse>
+    @type msgpack
+  </parse>
+  <format>
+    @type json
+  </format>
+</source>
+```
+
+But, we recomment to use in/out [`forward`](forward.md) plugin to communicate with two Fluentds due to at-most-once and at-least-once semantics for rigidty.
```
Comment (repeatedly) [inline plugins/output/http.md:366]: type: recommend

Comment (cosmo0920) [reply plugins/output/http.md:366]: Oops. 😖

Comment (cosmo0920) [reply plugins/output/http.md]: Fixed in https://github.com/fluent/fluentd-docs-gitbook/pull/184/commits/e2d21f1505a3246fb3d1602adba1b121372d500c.

**Comment by repeatedly** [comment]
Thanks! Awesome addition :)

**Pull request merged**